test/lib/usd/utils/CMakeLists.txt

```diff
@@ -93,8 +87,7 @@ mayaUsd_add_test(${TARGET_NAME}
 # -----------------------------------------------------------------------------
 
 set(USDUTILS_PYTHON_TEST_NAME Python:MayaTransformAPI)
-add_test(
-  NAME ${USDUTILS_PYTHON_TEST_NAME}
+mayaUsd_add_test(${USDUTILS_PYTHON_TEST_NAME}
   COMMAND
     python -m unittest discover -s ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
@robthebloke   Running python test from source code is not desirable. When these tests run they pollute the source code with bytecode ( *.pyc ) .

Merge dev into this branch and have a look at how UFE does it? We want to copy the any python codes and resources to the build directory and run the tests from there:

https://github.com/Autodesk/maya-usd/blob/819ce49d4bcfb50cbcd51f17adf2e2a3d5f30ea9/test/lib/ufe/CMakeLists.txt#L53

After the merge you will be able to set the working directory for the tests:

`WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}`